liquidator/liquidator.js

```diff
@@ -376,12 +403,37 @@ class Liquidator {
         continue;
       }
 
+      // Get resolved price request for dispute. This will fail if there is no price for the liquidation timestamp, which is possible if the
+      // liquidation expired without dispute.
+      let resolvedPrice;
+      if (requestTimestamp) {
+        try {
+          resolvedPrice = revertWrapper(
+            await this.votingContract.getPrice(this.empIdentifier, requestTimestamp, {
+              from: this.empContract.options.address
+            })
+          );
+          if (!resolvedPrice) {
+            // No price available for liquidation time, likely that liquidation expired without dispute.
+          }
+        } catch (error) {
+          // No price available for liquidation time, likely that liquidation expired without dispute.
+        }
+      }
+
       const logResult = {
         tx: receipt.transactionHash,
         caller: receipt.events.LiquidationWithdrawn.returnValues.caller,
         withdrawalAmount: receipt.events.LiquidationWithdrawn.returnValues.withdrawalAmount,
-        liquidationStatus: receipt.events.LiquidationWithdrawn.returnValues.liquidationStatus
+        liquidationStatus:
+          PostWithdrawLiquidationRewardsStatusTranslations[
+            receipt.events.LiquidationWithdrawn.returnValues.liquidationStatus
+          ]
       };
+      if (resolvedPrice) {
```

Comment:
Might make sense to push the comment from above down here?
```
// No price available for liquidation time, likely that liquidation expired without dispute.
```

Reply:
Yeah good call